app/models/katello/glue/pulp/repo.rb

```diff
@@ -775,6 +775,8 @@ def full_path(smart_proxy = nil)
       scheme   = (self.unprotected ? 'http' : 'https')
       if docker?
         "#{pulp_uri.host.downcase}:5000/#{pulp_id}"
+      elsif file?
+        "#{pulp_uri.host.downcase}/pulp/repos/#{pulp_id}"
```

Comment:
after playing around on my devbox, it looks like ```/pulp/isos/#{pulp_id}``` is correct.  I can browse to:

http://devbox/pulp/isos/Default_Organization-Red_Hat_Enterprise_Linux_Server-Red_Hat_Enterprise_Linux_6_Server_-_Supplementary_ISOs_i386_6_1/

for example